lib/routes/routePOST.js

```diff
@@ -2,13 +2,16 @@ import { errors } from 'arsenal';
 
 import api from '../api/api';
 import routesUtils from './routesUtils';
+import writeContinue from '../utilities/writeContinue';
 
 const MAX_POST_LENGTH = 1024 * 1024; // 1 MB
 
 /* eslint-disable no-param-reassign */
 export default function routePOST(request, response, log) {
     log.debug('routing request', { method: 'routePOST' });
 
+    // Send continue message, if applicable, for streaming V4 requests.
+    writeContinue(request, response);
```

Comment:
_Same suggestion_ here with `initiateMultipartUpload`, `completeMultipartUpload` and `multiObjectDelete`